toolbox/util/commonUtils.go

```diff
@@ -36,6 +36,9 @@ func ReadFile(filePath string) (string, error) {
 
 // WriteFile overwrites the file on the given path with content
 func WriteFile(filePath, content string) error {
```

Comment:
nit: might need to be changed to WriteTextFile if you assume you can add a \n
or are you sure it's never used to write binary data? (I guess it takes a string and not bytes so maybe that's implicit)

Reply:
Renaming to `WriteTextFile` totally makes sense. It is just difficult to know all the places this function has been used and also refactor every such occurrence. Maybe we could leave this text input as implicit defined as you have pointed out, and create a `WriteBinaryFile` when we need it.

Reply:
looks like it's used exactly once so far:
https://github.com/istio/test-infra/search?l=Go&q=WriteFile&type=&utf8=%E2%9C%93
so I would rename it now :-)

Reply:
Done